### PR TITLE
case DEV-492:  Ensure launcher cannot be pinned

### PR DESCRIPTION
**Pull request by amerhifi**
https://highfidelity.atlassian.net/browse/DEV-492

Preventing user from pinning HQ Launcher accidentally.

### CODE REVIEW

File: launchers/win32/LauncherUtils.cpp [hunk 1]
```diff
@@ -150,7 +150,8 @@ BOOL LauncherUtils::launchApplication(LPCWSTR lpApplicationName, LPTSTR cmdArgs)
     ZeroMemory(&si, sizeof(si));
     si.cb = sizeof(si);
     ZeroMemory(&pi, sizeof(pi));
-
+    
+    si.dwFlags = STARTF_TITLEISAPPID | STARTF_PREVENTPINNING;
```
Comment (huffman) [inline launchers/win32/LauncherUtils.cpp]: Would this present interface from being pinned?

Comment (amerhifi) [reply launchers/win32/LauncherUtils.cpp]: yes. but It did not during testing.  I didn't want to check for application name by string but I think that's the easiest option , as long as we don't change the name

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21120/HighFidelity-Beta-PR16127-08ca702.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21120/HighFidelity-Beta-Interface-PR16127-08ca702.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21120/HighFidelity-Beta-PR16127-08ca702.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21120/HighFidelity-Beta-Interface-PR16127-08ca702.dmg

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21120/HighFidelity-Beta-PR16127-08ca702.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21120/HighFidelity-Quest-Beta-PR16127-08ca702.apk)

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21125/HighFidelity-Beta-PR16127-fe3879d.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21125/HighFidelity-Beta-Interface-PR16127-fe3879d.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21125/HighFidelity-Beta-PR16127-fe3879d.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21125/HighFidelity-Beta-Interface-PR16127-fe3879d.exe

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21125/HighFidelity-Beta-PR16127-fe3879d.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21125/HighFidelity-Quest-Beta-PR16127-fe3879d.apk)

File: launchers/win32/LauncherUtils.cpp [hunk 2]
```diff
@@ -150,7 +150,7 @@ BOOL LauncherUtils::launchApplication(LPCWSTR lpApplicationName, LPTSTR cmdArgs)
     ZeroMemory(&si, sizeof(si));
     si.cb = sizeof(si);
     ZeroMemory(&pi, sizeof(pi));
-
+
```
Comment (amerhifi) [inline launchers/win32/LauncherUtils.cpp]: added the space back

**Comment by hifi-gustavo** [comment]
Android build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21166/HighFidelity-Beta-PR16127-9e69b79.apk).  Quest build is available [here](https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/21166/HighFidelity-Quest-Beta-PR16127-9e69b79.apk)

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21166/HighFidelity-Beta-PR16127-9e69b79.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/21166/HighFidelity-Beta-Interface-PR16127-9e69b79.exe

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21166/HighFidelity-Beta-PR16127-9e69b79.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/21166/HighFidelity-Beta-Interface-PR16127-9e69b79.dmg

**Pull request merged**